index.html

```diff
@@ -331,131 +322,220 @@
+          sandboxed top-level navigation browsing context flag</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/browsers.html#sandboxing-flag-set">sandboxing
+          flag set</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/browsers.html#session-history">session
+          history</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/webappapis.html#task-source">task
+          source</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/browsers.html#top-level-browsing-context">
+          top-level browsing context</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/infrastructure.html#trusted">trusted
+          event</a></dfn>
+        </li>
+        <li>
+          <dfn><a href=
+          "https://www.w3.org/TR/html51/browsers.html#unloaded">unload a
+          document</a></dfn>
+        </li>
+      </ul>
       <p>
         The term <dfn><a href=
         "https://tc39.github.io/ecma262/#sec-code-realms">JavaScript
         realm</a></dfn> is defined in [[!ECMASCRIPT]].
       </p>
       <p>
-        The terms <dfn><code><a href=
-        "https://dom.spec.whatwg.org/#eventtarget">EventTarget</a></code></dfn>,
-        <dfn><code><a href=
-        "https://dom.spec.whatwg.org/#event">Event</a></code></dfn>,
-        <dfn><code><a href=
-        "https://dom.spec.whatwg.org/#dictdef-eventinit">EventInit</a></code></dfn>
+        The terms <a href=
+        "https://dom.spec.whatwg.org/#eventtarget"><dfn><code>EventTarget</code></dfn></a>,
+        <a href=
+        "https://dom.spec.whatwg.org/#event"><dfn><code>Event</code></dfn></a>,
+        <a href=
+        "https://dom.spec.whatwg.org/#dictdef-eventinit"><dfn><code>EventInit</code></dfn></a>
         are defined in [[!DOM]].
       </p>
       <p>
-        The term <dfn><code><a href=
-        "http://www.w3.org/TR/webmessaging/#messageevent">MessageEvent</a></code></dfn>
+        The term <a href=
+        "https://www.w3.org/TR/webmessaging/#messageevent"><dfn><code>MessageEvent</code></dfn></a>
         is defined in [[!WEBMESSAGING]].
       </p>
       <p>
         This document provides interface definitions using the Web IDL standard
-        ([[!WEBIDL]]). The terms <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-promise">Promise</a></dfn>,
-        <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
-        <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>
-        are defined in [[!WEBIDL]].
+        [[!WEBIDL-1]].
       </p>
       <p>
         The term <dfn><a href=
         "https://heycam.github.io/webidl/#dfn-throw">throw</a></dfn> in this
```

Comment:
It seems like _throw_ could be included in the list in the following paragraph.  I can change this separately.